src/ExponentialMap.jl

```diff
@@ -311,7 +311,8 @@ exponential, and ``X`` is a set, it follows that
 """
 function σ(d::AbstractVector{N},
            eprojmap::ExponentialProjectionMap)::AbstractVector{N} where {N<:Real}
-    daux = transpose(eprojmap.projspmexp.L) * d
+    d_dense = d isa Vector ? d : Vector(d)
+    daux = transpose(eprojmap.projspmexp.L) * d_dense
```

Comment:
maybe add a comment `# expmv requires a Vector`

Reply:
ah, just saw the comment in the docstring above :)

Reply:
But I forgot it in the second case 👍